src/client/app/requests.js

```diff
@@ -11,13 +11,18 @@ export const Urls = {
     return url
   },
 
-  File: name => `/files/${name}`,
+  File: name => new URL(`files/${name}`, window.location.href).href,
 
   JWTFile: (name, filesEndpoint) => `${filesEndpoint}/rtc-visualizer/files/${name}`,
```

Comment:
the JWTFile function will no longer require the filesEndpoint, (this was received via the config request which we removed). Please remove the filesEndpoint and check where the JWTFIle request is made from and adjust accordingly.